src/Workspaces/Core/Portable/xlf/WorkspacesResources.es.xlf

```diff
@@ -7,6 +7,11 @@
         <target state="translated">Error al leer el archivo de configuración especificado: {0}</target>
         <note />
       </trans-unit>
+      <trans-unit id="AsyncLazy_StaticInit_InvalidOperation">
+        <source>ValueFactory returned null.</source>
```

Comment:
it's a bit odd since hte parameter name is actually 'valueFactory'.  Perhaps `'valueFactory' returned null`?

Reply:
Or "The value factory returned null."